R/derive_extreme_event.R

```diff
@@ -23,20 +34,51 @@
 #'
 #'   *Permitted Values:* `"first"`, `"last"`
 #'
-#' @param events Conditions and new values defining events
+#' @param source_datasets Source datasets
+#'
+#'   A named list of datasets is expected. The `dataset_name` field of `event()`
+#'   and `event_joined()` refers to the dataset provided in the list.
+#'
+#' @param ignore_event_order Ignore event order
+#'
+#'   If the argument is set to `TRUE`, all events defined by `events` are
+#'   considered equivalent. If there is more than one observation per by group
+#'   the first or last (with respect to `mode` and `order`) is select without
+#'   taking the order of the events into account.
```

Comment:
Should we say this is set to FALSE by default?

Reply:
According to our convention we should not state the default value in the description.

Reply:
ah right - i should probably internalize this convention!